bitrise.yml

```diff
@@ -1062,9 +1063,9 @@ workflows:
                 set -x
 
                 mkdir -p /bitrise/src/trace-test-application/build/reports
-                adb logcat -d > $BITRISE_SOURCE_DIR/trace-test-application/build/reports/logcat.txt &
+                adb logcat -d > $BITRISE_SOURCE_DIR/trace-test-application/build/reports/logcat.txt
```

Comment:
There is no guarantee that it will be finished till the deploy of the reports happen, so decided to make it synchronous instead of async.